### PR TITLE
feat: update and fixes

### PR DESCRIPTION
**Pull request by nasbench**
### Summary of the Pull Request

This PR is part of the major SIGMA fix. It updates and fixes multiple rules.

### Detailed Description of the Pull Request / Additional Comments

Apart from obvious changes to file names and rule titles. I updated the allowed rule title length to 100. Because it 70 is just too short for some rules and it's always better to be clear than short.

### Example Log Event

N/A

### Fixed Issues

N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_sc_stop_service.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+title: Stop Windows Service Via Sc.EXE
+id: 81bcb81b-5b1f-474b-b373-52c871aaa7b1
+related:
+    - id: eb87818d-db5d-49cc-a987-d5da331fbd90
+      type: obsoletes
+status: experimental
+description: Detects a windows service to be stopped
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_sc_stop_service.yml:7]:
```suggestion
description: Detects the stopping of a Windows service

File: rules/windows/process_creation/proc_creation_win_susp_execution_from_guid_folder_names.yml [hunk 2]
```diff
@@ -37,5 +37,5 @@ detection:
         Image: 'C:\Windows\System32\drvinst.exe'
     condition: all of selection_* and not 1 of filter*
 falsepositives:
-    - Some FP is expected with some installers
+    - Installers are sometimes known for creating temporary folders with GUID like names. Add apropriate Filters accordingly
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_susp_execution_from_guid_folder_names.yml:40]:
```suggestion
    - Installers are sometimes known for creating temporary folders with GUID like names. Add appropriate filters accordingly
```

File: rules/windows/process_creation/proc_creation_win_susp_service_tamper.yml [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+id: ce72ef99-22f1-43d4-8695-419dcb5d9330
+related:
+    - id: eb87818d-db5d-49cc-a987-d5da331fbd90
+      type: derived
+    - id: 6783aa9e-0dc3-49d4-a94a-8b39c5fd700b
+      type: obsoletes
+    - id: 7fd4bb39-12d0-45ab-bb36-cebabc73dc7b
+      type: obsoletes
+status: experimental
+description: Detects the usage binaries such as 'net', 'sc' or 'powershell' in order to stop, pause or delete critical or important windows services such as AV, Backup...etc. As seen being used in some ransomware scripts
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_susp_service_tamper.yml:11]:
```suggestion
description: Detects the usage of binaries such as 'net', 'sc' or 'powershell' in order to stop, pause or delete critical or important Windows services such as AV, Backup, etc. As seen being used in some ransomware scripts
```

File: rules/windows/process_creation/proc_creation_win_susp_service_tamper.yml [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+            - 'WdNisSvc'
+            - 'WeanClOudSve'
+            - 'Weems JY'
+            - 'WinDefend'
+            - 'wozyprobackup'
+            - 'WRSVC'
+            - 'Zoolz 2 Service'
+    condition: selection_services and (all of selection_net_* or all of selection_pwsh_* or all of selection_sc_*)
+falsepositives:
+    - Administrator or tools shutting down the services due to upgrade or removal purposes. If you experience some FP please consider adding filters to the parent process launching this command and not removing the entry
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_susp_service_tamper.yml:237]:
```suggestion
    - Administrators or tools shutting down the services due to upgrade or removal purposes. If you experience some false positive, please consider adding filters to the parent process launching this command and not removing the entry
```

**Review by phantinuss — APPROVED** [review]

File: rules/windows/process_creation/proc_creation_win_hktl_powersploit_empire_default_schtasks.yml [hunk 5]
```diff
@@ -29,17 +29,14 @@ detection:
         Image|endswith: '\schtasks.exe'
         CommandLine|contains|all:
             - '/Create'
-            - '/SC'
-            - '/TN'
-            - 'Updater'
-            - '/TR'
-            - 'powershell'
+            - 'powershell.exe -NonInteractive'
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_hktl_powersploit_empire_default_schtasks.yml:32]:
```suggestion
            - 'powershell.exe -NonI'
```
https://github.com/EmpireProject/Empire/blob/08cbd274bef78243d7a8ed6443b8364acd1fc48b/lib/modules/powershell/persistence/userland/schtasks.py#L219

**Pull request merged**